src/main/java/com/jcabi/dynamodb/maven/plugin/AbstractDynamoMojo.java

```diff
@@ -96,6 +96,8 @@ public void execute() throws MojoFailureException {
             Logger.info(this, "execution skipped because of 'skip' option");
             return;
         }
+        // @checkstyle Always make this.environment(); as the first call!
```

Comment:
@simonjenga why do you use `@checkstyle` annotation here?
if you really need this comment, put it into javadoc